site/docs/workflows/dart_package.md

```diff
@@ -103,6 +103,12 @@ The Dart package workflow consists of the following steps:
 
 **Default** `false`
 
+### `no_example`
+
+**Optional** To avoid getting packages on an `example/` when running `dart pub get`.
```

Comment:
Nit: I would add the clarification that the `example` command has on its [help](https://github.com/dart-lang/pub/blob/61716827b14e1848d82f1070ed5be881650c511d/lib/src/command/get.dart#L54C14-L54C52).

```suggestion
**Optional** To avoid getting packages in `example/` when running `dart pub get` (if it exists).
```